docs/server/configuration/security.md

```diff
@@ -722,32 +722,87 @@ programmatically.
 
 This allows administrators to define stream access policies for EventStoreDB based on stream prefix.
 
-### Configuration
+### Enabling
 
 You require a [license key](../quick-start/installation.md#license-keys) to use this feature.
 
-Refer to the [configuration guide](../configuration/README.md) for configuration mechanisms other than YAML.
+The stream access policy is configured by the last event in the `$authorization-policy-settings` stream.
 
-The feature can be enabled by setting the `Authorization:PolicyType` option to `streampolicy`.
+Enable stream policies by appending an event to the `$authorization-policy-settings` stream:
 
-You can do this by adding the following to the server configuration file:
+```json
```

Comment:
Should I include the HTTP requests or curl commands to append these events?

Reply:
@hayley-jean Is there a reason to "not" add them? Seems like relevant information - but perhaps I'm missing something.

Reply:
Good point, I've added them :)